server.py

```diff
@@ -10,7 +10,17 @@
 @app.route('/')
 def home_page():
     now = datetime.datetime.now()
-    return render_template('home.html', current_time=now.ctime())
```

Comment:
Why do you changed route page? I supposed to do the home page so do I need to change home.html to index.html?

Reply:
Hello there Bekci,
Yes

Reply:
Yes we have changed organization of folders and html pages our homepage is now index.html .You should change it as well.

Reply:
Okay everything is clear now. Thanks!